plugins/package-managers/spm/src/funTest/assets/projects/synthetic/spm-app/Package.resolved

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
@UgniusV  I'm trying to enhance the implementation and I'm surprised by the entries in this lockfile.
Can you please explain how this file has been created? 

In particular, has the file been generated by the `swift` command? Has it been modified or been crafted by hand?
(I'm after the combination of `branch` `revision` `version`. E.g. which ones can be present. After playing around a while with `spm` I was unable to make it generate anything like this.

Reply:
If you could provide instructions how to reproduce the creation steps of this file...